File: tests/loops.py
for i in range(5):
    print("for", i)

count = 0
while count < 5:
    print("while", count)
    count += 1

for i in range(1, 4):
    for j in range(1, 4):
        if i != j:
            print(f"({i}, {j})")

File: tests/loops.go
package main

import "fmt"

func main() {
	for i := 0; i < 5; i++ {
		fmt.Println("for", i)
	}

	count := 0
	for count < 5 {
		fmt.Println("while", count)
		count++
	}

	for i := 1; i < 4; i++ {
		for j := 1; j < 4; j++ {
			if i != j {
				fmt.Printf("(%v, %v)\n", i, j)
			}
		}
	}
}
